### DISCUSSION TITLE
End of life for non-HTTPS access

### DISCUSSION THREAD

**Discussion by joeltankam** [NuGet/Home#11947] [Q&A]
There's currently a warning when using a non-HTTPS feed for NuGet restore:

> NU1803: You are running the 'restore' operation with an 'HTTP' source, '...'. Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS' source.

This was introduced in https://github.com/NuGet/NuGet.Client/pull/4556

Could you please give the exact release (Visual Studio in particular) for which this support will be removed ?

Ping @nkolev92 @kartheekp-ms @davidcorrigan714

**Comment by kartheekp-ms** [comment]
Please refer to https://devblogs.microsoft.com/nuget/https-everywhere/ blog for more information.

**Reply by shawty** [reply]
VERY BAD IDEA MS... please read the comments on the blog post you point too....

There are many reasons to allow HTTP access.
